### DISCUSSION TITLE
Code doesnt Work!

### DISCUSSION THREAD

**Discussion by RobertArustamyan** [community/community#112275] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

import keyboard
from Functions.getting_random_words import  get_word
from Functions.loading_data_per_page import read_load_json_file

# keyboard.wait('enter')
# print("Word: XXX", end="", flush=True)
# keyboard.wait('enter')
# print("Translation ", end="", flush=True)

if __name__ == "__main__":
Keyboard command doesnt work

**Comment by SergeyArzumanyan** [comment, marked as answer]
main doesn't contain anything )

**Comment by RobertArustamyan** [comment]
It worked

**Comment by ghostinhershell** [comment]
Please see my response here: https://github.com/orgs/community/discussions/112279#discussioncomment-8765821
